wiki/People/The_Team/Beatmap_Nominators/en.md

```diff
@@ -82,6 +82,7 @@ The tables listed below indicate the game mode(s) in which each Beatmap Nominato
 | ![][flag_KR] [Riana](https://osu.ppy.sh/users/1997633) | Korean |
 | ![][flag_NZ] [Rizen](https://osu.ppy.sh/users/3056732) |  |
 | ![][flag_SG] [Sinnoh](https://osu.ppy.sh/users/4236057) |  |
+| ![][flag_SG] [Smoke](https://osu.ppy.sh/users/10726630) |  |
```

Comment:
wrong flag i think? looks like he's from the US

Reply:
fixed

Reply:
fixed!